### PR TITLE
Implementation of styles using modules in Feature/styles

### PR DESCRIPTION
**Pull request by Estete9**
### Project requirements
- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in [Zeplin](https://zeplin.io/) to style the website.

### CODE REVIEW

File: src/styles/header.module.css [hunk 1]
```diff
@@ -15,8 +20,21 @@
...
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
+}
+
+.profile img {
+  height: 2.813rem;
+}
+
 .navbarItem > * {
```
Comment (nahnahdev) [inline src/styles/header.module.css:34]: - [ ] Great job so far @Estete9 There are a few changes I will suggest you implement and you'll be on your way to the next milestone 😉 
- [x] Your `Navbar` currently has a background color different from that of the given Zeplin design so kindly fix that 👍
- [x] You can also see that your font size for `Books and Categories` looks a little smaller and there are no spaces between the letters as seen in the Zeplin design so kindly fix that as well 😉 
- Your Design
image

- The Zeplin Design
image

File: src/components/Book.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import styles from '../styles/book.module.css';
 import { removeBookAPI } from '../redux/features/bookstore/BooksSlice';
+import Button from './Button';
+import progressImg from '../assets/progressBar.png';
 
 function Book({ id, title, author }) {
```
Comment (nahnahdev) [inline src/components/Book.js:9]: - [x] Currently, there is a gray border present in the Zeplin design that is absent in yours so kindly include that as well 👍 
- [x] From the images below, you can see that the size of your card is a little smaller than that of the Zeplin design. So kindly fix that 👍 
- [x] Kindly ensure that the `author name`, `remove`, `comment`, `edit` and  `completed` buttons/texts have a similar font size, font-weight and color as seen in the Zeplin design 😉 
- Your Design
image

- The Zeplin Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Estete9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 🔥 
- All elements in the design are added ✔️ 
- Linters well setup and passing ✔️ 
- Professional README ✔️ 
- Nice design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @Estete9 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats
## Highlights

- [ ] Apply React best practices and language style guides in code. 👍
- [ ] Use Professional Readme file 👍
- [ ] Descriptive PR Summary used 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alsup in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**